content/en/docs/tasks/security/authorization/authz-jwt/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Authorization for JWT
+description: Shows how to set up access control with JWT in Istio.
+weight: 30
+keywords: [security,authorization,jwt,claim]
+aliases:
+    - /docs/tasks/security/rbac-groups/
+    - /docs/tasks/security/authorization/rbac-groups/
+---
+
+This task shows you how to set up an Istio authorization policy to enforce access
+based on the JSON Web Token (JWT). An Istio authorization policy supports both string typed
+and list-of-string typed JWT claims.
+
+## Before you begin
+
+Before you begin this task, perform the following actions:
+
+* Read the [Authorization](/docs/concepts/security/#authorization) and [Authentication](/docs/concepts/security/#authentication).
+
+* Install Istio using the [Istio installation guide](/docs/setup/install/istioctl/).
+
+* Deploy two workloads: `httpbin` and `sleep`. Deploy these in one namespace,
+for example `foo`. Both workloads run with an Envoy proxy in front of each.
+Deploy the example namespace and workloads using these commands:
+
+    {{< text bash >}}
+    $ kubectl create ns foo
+    $ kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml) -n foo
+    $ kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml) -n foo
+    {{< /text >}}
+
+* Verify that `sleep` successfully communicates with `httpbin` using this command:
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output, retry after a few seconds.
+Caching and propagation can cause a delay.
+{{< /warning >}}
+
+## Allow requests with valid JWT and list-typed claims
+
+1. The following command creates the `jwt-example` request authentication policy
+for the `httpbin` workload in the `foo` namespace. The policy allows the Envoy proxy in front of
+the `httpbin` workload to accept a JWT issued by `testing@secure.istio.io`.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: "security.istio.io/v1beta1"
+    kind: "RequestAuthentication"
+    metadata:
+      name: "jwt-example"
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      jwtRules:
+      - issuer: "testing@secure.istio.io"
+        jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+    EOF
+    {{< /text >}}
+
+1. Verify that a request with an invalid JWT is denied.
```

Comment:
numbered list instructions should typically end with a `:`. In this case, change `denied.` to -> `denied:`